one-pager-maker/src/pages/List.tsx

```diff
@@ -132,7 +132,7 @@ function DocumentListSection({
       {status === "error" && <ErrorContainer>{error?.message}</ErrorContainer>}
       {status === "pending" && (
         <Grid>
-          {[0, 0, 0].map((_) => (
+          {[0, 1, 2].map((_) => (
```

Comment:
What is this?

Reply:
There was an rendering error because key of Grid Component was indentical. 
That's why I changed values of key to have different values.

Reply:
Maybe it's just a meaningless list to begin with? It's OK for now.